src/libostree/ostree-fetcher-soup.c

```diff
@@ -360,6 +362,13 @@ session_thread_set_tls_database_cb (ThreadClosure *thread_closure, gpointer data
     }
 }
 
+static void
+session_thread_set_max_outstanding_fetcher_requests (ThreadClosure *thread_closure, gpointer data)
+{
+  guint32 n = GUINT_TO_POINTER (data);
+  thread_closure->opt_max_outstanding_fetcher_requests = n;
```

Comment:
Here we want to go the other way so try `GPOINTER_TO_UINT (data)`; how about just:
```suggestion
  thread_closure->opt_max_outstanding_fetcher_requests = GPOINTER_TO_UINT (data);
```
?

Reply:
thanks!